introduction.md

```diff
@@ -0,0 +1,2 @@
+Hello Everyone, My name is Joffre and I'm practicing the excercises for GIT from Code School.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.